src/index.js

```diff
@@ -1,7 +1,7 @@
 import './style.css';
+import addCheck from './status.js';
 
-const tasksList = [];
-let ids = 0;
+let tasksList = [];
```

Comment:
- [ ] It would be better if we have a default task so we can check the following:
  - [ ] The task object's value for `completed` is updated upon user actions
  - [ ] The updated array of items is stored in local storage